external/craigslist/fetch.py

```diff
@@ -2,6 +2,6 @@
 from typing import Dict, List
 
 
-def fetch_craigslist_housing(*, limit, **kwargs) -> List[Dict[str, any]]:
+def fetch_craigslist_housing(**kwargs) -> List[Dict[str, any]]:
```

Comment:
I know this is a draft PR but just to make sure, let me leave a note. 
This needs to be restored in the final PR.

Reply:
hahaha, I'm thinking about removing it for good.
Is there any potential risk to remove it?

Reply:
haha yes it's intended.

It's easy for people to omit the limit parameter and omitting it has a potential risk of receiving a large size of response from craigslist, which is also very slow and not good.
That's why I set limit as a required parameter in order to make people aware of this.